source/part1/chapter-01/md/00-motivation.md

```diff
@@ -0,0 +1,93 @@
+029740 99999 195201020000 220   7 *** 722 CLR 0 0 0 12.5 02 ** ** ** ** ** ** ** 0   36   28  995.8 ***** ****** *** *** ***** ***** ***** ***** **
+029740 99999 195201020600 220  16 ***  15 OVC 5 * *  9.4 02 ** ** ** ** ** ** ** 1   36   34  997.1 ***** ****** ***  34 ***** ***** ***** ***** **
+029740 99999 195201021200 110  14 ***   8 OVC 5 * * 12.5 70 ** ** ** ** ** ** ** 2   34   28  993.1 ***** ****** *** *** ***** ***** ***** ***** **
+029740 99999 195201021800 160  14 ***   8 OVC 7 * *  1.2 73 ** ** ** ** ** ** ** 7   34   32  985.9 ***** ******  37 *** ***** ***** ***** ***** **
+029740 99999 195201030000 180  18 ***  15 OVC 5 * *  3.8 26 ** ** ** ** ** ** ** 7   36   28  985.4 ***** ****** *** *** ***** ***** ***** ***** **
+029740 99999 195201030600 200  14 ***  15 BKN 5 * *  5.0 02 ** ** ** ** ** ** ** 7   36   32  985.2 ***** ****** ***  32 ***** ***** ***** ***** **
+...
+```
+
+Not that exciting, right? There is an interesting story here, but we need some way to illustrate the power of this data.
+
+One option is to use an *x*-*y* plot of temperature anomalies versus time.
+
+![Global mean temperature anomalies](https://www.ncdc.noaa.gov/sotc/service/global/global-land-ocean-mntp-anom/201101-201112.png)
+
+**Figure 1.1**. Global mean temperature anomalies from 1880-2011. Source: <https://www.ncdc.noaa.gov/sotc/global/201113>.
+
+This is obviously much better, nicely showing how temperatures have changed with time and how global temperatures have increased significantly since 1970. Now we have taken a clear step toward making the data easier to understand. However, these are global data and we are missing something important about them, their connection to geographical locations.
+
+Let's consider another option, plotting temperature anomalies on a map.
+
+![Global mean temperature anomaly map](https://www.ncdc.noaa.gov/sotc/service/global/map-blended-mntp/202001.png)
+
+**Figure 1.2**. Global temperature anomalies for January 2020. Source: <https://www.ncdc.noaa.gov/sotc/global/201603>.
+
+And yet again, this helps us understand the data further. Not only do we see the changes in temperature, but now we see how temperatures vary in space across the globe. The drawback here is that we only see a single time snapshot, rather than a time series. Seeing both will require a truly remarkable visualization.
+
+So, let's look now at some excellent examples of data visualization with Python. We have essentially the same data plotted above, but now we can see how temperatures vary in space and time.
+
+```python tags=["hide-input"]
+Video('../../../_static/Temp-anomalies-2018.mp4', width=800)
+```
+
+**Figure 1.3**. Global temperature anomalies by country from 1900-2017. Visualization by Antti Lipponen ([@anttilip](https://twitter.com/anttilip) [^antti_twitter]). The animation can be found online at <https://flic.kr/p/293M1oa>.
```

Comment:
We need to figure out what to do with this video. Unfortunately renders poorly in a book 😛

Reply:
Yes, I agree. Perhaps we can create some still images that can be used as links to the video on the website, but show up in a reasonable way in the book?

Reply:
Yes, that is one possibility. 👍 Another option I guess would be to think changing the content a bit so that it wouldn't refer to an animation but something else, but that is not very good either because one of the main idea in automation is to be able to repeat analyses and an animation is a perfect example of such.. So I guess a still image of the video and a footnote with link would be the best way forward.

Reply:
I've now tweaked things to use images of the videos on the website with links to the videos on flickr. I suppose this works for the website and book, and things mostly look the same overall. What do you think @HTenkanen ?

Reply:
This was a bit more than could be done easily in the GitHub interface, so I committed and pushed my local changes.